packages/bootstrap-4/src/DescriptionField/DescriptionField.tsx

```diff
@@ -1,24 +1,22 @@
-import React from 'react';
+import React from "react";
 
-import { FieldProps } from '@rjsf/core';
+interface DescriptionFieldI  {
```

Comment:
I'm not a massive fan of the `I` suffix on interfaces. Personally, I'd call this type `DescriptionFieldProps`.

There's also a double space before the curly brace. Could we standardise on running Prettier on the code before pushing? I personally have `Format on save` setting turned on in my VSCode, which does the job.

Reply:
> I'm not a massive fan of the `I` suffix on interfaces. Personally, I'd call this type `DescriptionFieldProps`.

Cool, I will change that. 




> There's also a double space before the curly brace. Could we standardise on running Prettier on the code before pushing? I personally have `Format on save` setting turned on in my VSCode, which does the job.

I do have prettier installed, but idk I might have miss configured it to align with `.tsx` files. I will check that.